app.py

```diff
@@ -355,6 +376,31 @@ def main():
 	"""Redirect to main page."""
 	return redirect(url_for('login'))
 
+@app.route('/annotate/direct_entry', methods=['GET'])
+@loginrequired
+def direct_entry():
+	"""Directly enter a sentence."""
+	sent = request.args.get('sent', '').strip()
+	sentid = str(request.args.get('id', '')).strip()
+	if len(sent) == 0:
+		return jsonify({'error': 'Sentence is empty.'})
+	elif len(sentid) == 0:
+		return jsonify({'error': 'Sentence ID is empty.'})
+	db = getdb()
+	cur = db.execute(
+		'SELECT * FROM entries ORDER BY sentno ASC'
```

Comment:
If you are going to be accessing fields for each record by index (`entry[0]`) probably best to specify the columns by name.